stl/inc/utility

```diff
@@ -80,29 +80,14 @@ _CONSTEXPR20 void iter_swap(_FwdIt1 _Left, _FwdIt2 _Right) { // swap *_Left and
 
 template <class _Ty, size_t _Size, enable_if_t<_Is_swappable<_Ty>::value, int> _Enabled>
 _CONSTEXPR20 void swap(_Ty (&_Left)[_Size], _Ty (&_Right)[_Size]) noexcept(_Is_nothrow_swappable<_Ty>::value) {
-    _Ty* _First1 = _Left;
-    _Ty* _Last1  = _First1 + _Size;
-    _Ty* _First2 = _Right;
-
-    if (_First1 == _First2) {
-        return;
-    }
-
-#if _USE_STD_VECTOR_ALGORITHMS
-    if constexpr (_Is_trivially_swappable_v<_Ty>) {
-#if _HAS_CXX20
-        if (!_STD is_constant_evaluated())
-#endif // _HAS_CXX20
-        {
-            __std_swap_ranges_trivially_swappable_noalias(_First1, _Last1, _First2);
-            return;
+    if (&_Left != &_Right) {

```

Comment:
Just noting that I'm not going to complain about this hijackable use of `&` since I know we intend to re-apply the change here once we've addressed the logistics. It's been here for many years without any user complaints, presumably another point release cycle won't be problematic. (No change requested.)

Reply:
I noticed that too (one reason I had changed the code) although technically I believe that `&array` isn't hijackable the way that `&array[0]` is.

Reply:
After reviewing [basic.lookup.argdep], I agree. I was thinking that the associated entities of an array type should include the associated entities of its element type - which seems consistent with the rule for pointers - but that is NOT the case.

Nevermind.